README.md

```diff
@@ -14,9 +14,17 @@ It's responsible for:
 
 ## Requirements
 
+### Production Environment
+
 - Node.js version 20.x or higher
 - PostgreSQL
-- Docker and Docker Compose (optional, for containerized development)
+
+### Development Environment
+
+Either:
+
+- **Option 1:** Node.js + Docker and Docker Compose (recommended, handles PostgreSQL automatically)
```

Comment:
personal preference, but I don't think we should recommend docker.  while it does "solve" pg, it also introduces a lot of complexity with people not being able to access the app directly and needing to recreate it frequently.

Personally  `brew install postgres` seems _much_ simpler than dealing with docker all the time 😁

Reply:
I will make it more neutral. I think that's more in line with the both options explained further down the README.